docs/github.rst

```diff
@@ -3,6 +3,14 @@ GitHub
 
 `GitHub <https://github.com>`_ is an online tool for software collaboration.
 
+This ``GitHub`` class uses the `PyGitHub library <https://pygithub.readthedocs.io/en/latest/introduction.html>`_
+to make requests to the `GitHub REST API <https://docs.github.com/en/rest>`_. The class provides methods to:
+
+- get an individual user, organization, repo, issue, or pull request
+- get lists of user or organization repos for a given `username` or `organization_name`
+- get lists of repo issues, pull requests, and contributors for a given `repo_name`
+- download files and tables
```

Comment:
Possible to uppercase these? Also, I think that you need two ticks (``) for it to render as code.